app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,12 @@
   </head>
   <body class="<%= controller_name %> <%= action_name %> <%= if educator_signed_in? then "educator-signed-in" end %>">
     <div class="nav">
+      <% if ENV['DEMO_SITE_WARNING'] === 'SHOW' %>
```

Comment:
Alternately I think there's a `DEPLOYMENT_KEY` value or something like that already set that's used to indicate that it's the demo site.

Reply:
Ooooooh! I thought we had something like this but I couldn't remember what it was called! Thank you @kevinrobinson!